pkg/util/cgroups/cgroupv2_cpu.go

```diff
@@ -98,21 +103,21 @@ func parseV2CPUStat(stats *CPUStats) func(key, value string) error {
 	}
 }
 
-func parseV2CPUMax(stats *CPUStats) func(key, value string) error {
-	return func(limit, period string) error {
-		periodVal, err := strconv.ParseUint(period, 10, 64)
+func parseV2CPUMax(stats *CPUStats) func(key, value []byte) error {
+	return func(limit, period []byte) error {
+		periodVal, err := strconv.ParseUint(string(period), 10, 64)
```

Comment:
could you add `stringLimit` and `stringPeriod` to avoid doing 2 conversions from []byte to string 🙇